src/components/shared/SearchInput.js

```diff
@@ -38,6 +38,11 @@ class SearchInput extends Component {
 
   static defaultProps: Object;
 
+  componentDidMount() {
+    this.input.focus();
+    this.input.select();
```

Comment:
In the old code, we used setSelectionRange -- was this to move the cursor to the end of the input? I also noticed that in both cases we only fire select when the component is opened, and in that case it is empty. perhaps we don't need this?